tests/e2e/common-operator-integ-suite.sh

```diff
@@ -191,18 +214,50 @@ if [ "${SKIP_BUILD}" == "false" ]; then
     get_internal_registry
   fi
 
-  # BUILD AND PUSH IMAGE
+  # BUILD AND PUSH OPERATOR IMAGE
   build_and_push_image
 
-  if [ "${OCP}" == "true" ]; then
-    # This is a workaround
-    # To avoid errors of certificates meanwhile we are pulling the operator image from the internal registry
-    # We need to set image $HUB to a fixed known value after the push
-    # This value always will be equal to the svc url of the internal registry
-    HUB="image-registry.openshift-image-registry.svc:5000/sail-operator"
+  # If OLM is enabled, deploy the operator using OLM
+  if [ "${OLM}" == "true" ]; then
+    # Set the platform to Kubernetes by default.
+    # We are skipping the deploy via OLM test on OCP because the workaround to avoid the certificate issue is not working.
+    HELM_PLATFORM="kubernetes"
+    
+    # Install OLM in the cluster because it's not available by default in kind.
+    ${OPERATOR_SDK} olm install
+
+    # Set image-related variables
+    IMAGE_TAG_BASE="${HUB}/${IMAGE_BASE}"
```

Comment:
Why setting `IMAGE_TAG_BASE` here? Isn't that already defined from invocation of `make test.e2e.kind`

Reply:
For some reason when we invoke inside the script is not sharing that previous variable declaration when the first target was called. That is why I added again the set of variable

Reply:
But you are setting `IMAGE_TAG_BASE` which is not available from parent Makefile invocation but at the same time you expect that `TAG` is available. Or at least I don't see you define `TAG` anywhere.

Reply:
If I do this instead of setting the `IMAGE_TAG_BASE` to set later `BUNDLE_IMG` we get errors because `BUNDLE_IMG` is not correctly filled:

```
    HUB="${HUB}" \
    IMAGE_BASE="${IMAGE_BASE}" \
    OPENSHIFT_PLATFORM="false" \
    make bundle bundle-build bundle-push
```
When bundle-push is run we got that the `BUNDLE_IMG` is being filled with the default value:
```
docker build -f bundle.Dockerfile -t quay.io/maistra-dev/sail-operator-bundle:v0.1.0 .
[+] Building 2.1s (7/7) FINISHED                                                                                                                            docker:default
 => [internal] load build definition from bundle.Dockerfile                                                                                                           
```

This is because `IMAGE_TAG_BASE` and `BUNDLE_IMG` previously existed when the first target is getting invoked. So we need to set those values  manually inside the script

Reply:
`TAG` has the default value from the first make target invocation also, that is Why this value is filled correctly

Reply:
This type of issue we already have it before and is because of the circle invocation of the make targets. These variables are not going to be filled again in the make target because in the first invocation, they already filled:
```
# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
# quay.io/sail-dev/sail-operator-bundle:$VERSION and quay.io/sail-dev/sail-operator-catalog:$VERSION.
IMAGE_TAG_BASE ?= ${HUB}/${IMAGE_BASE}

BUNDLE_MANIFEST_DATE := $(shell cat bundle/manifests/${OPERATOR_NAME}.clusterserviceversion.yaml 2>/dev/null | grep createdAt | awk '{print $$2}')

# BUNDLE_IMG defines the image:tag used for the bundle.
# You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
```